docker-compose-ci.yml

```diff
@@ -0,0 +1,5 @@
+metrics:
```

Comment:
Is this file being used?

The `.prxci` file doesn't specify it, i.e. it would need to be:
```
docker-compose -f docker-compose-ci.yml build
```

so I think the CICD build is just using the regular default `docker-compose.yml`, and you can delete this file.

Reply:
You're probably right, but the guide says I should have both docker-compose.yml and docker-compose-ci.yml https://github.com/PRX/internal/pull/108/files#diff-5b3ca927271ab217a5d8c663b572f159R18

Reply:
Yeah, it's optional, for cases where you need the ci to do something slightly differently.
For example, we don't start up the worker for testing cms or feeder, so those get left out for ci.